server/chat-commands/core.ts

```diff
@@ -1068,7 +1068,12 @@ export const commands: ChatCommands = {
 	restoreplayershelp: [
 		`/restoreplayers - Restore previous players in an imported input log.`,
 	],
-
+	rejoinbattle(target, room, user) {
+		if (!user.rejoinBattle()) return this.errorReply(`You do not have a battle you can rejoin.`);
```

Comment:
This probably shouldn't be a command at all; it should just happen automatically, right? Maybe have it as part of `/autojoin`.